docs/content/usage/merge-message-templates.en-us.md

```diff
@@ -44,6 +44,8 @@ You can use the following variables enclosed in `${}` inside these templates whi
 - PullRequestIndex: Pull request's index number
 - PullRequestReference: Pull request's reference char with index number. i.e. #1, !2
 - ClosingIssues: return a string contains all issues which will be closed by this pull request i.e. `close #1, close #2`
+- ReviewedOnTrailer: Message trailer including pull request URL, i.e. `Reviewed-on: https://gitea.com/foo/bar/pulls/1`
+- ReviewedByTrailer: Message trailer including pull request approvers, i.e.,`Reviewed-by: Jane Doe <jane.doe@example.com>`
```

Comment:
Hmm… I'm not sure about the `-trailer` suffix as that isn't exactly helpful for understanding what it does.
Let's keep it a bit more context free and do either of the following:
- Remove the suffix completely (`{ReviewedOn}`, `{ReviewedBy}`)
- Use a different suffix such as `-Messages` or `-Lines`

My preference is removing the suffix.

Reply:
Additionally,
```suggestion
- ReviewedOn: Which pull request this commit belongs to. For example `Reviewed-on: https://gitea.com/foo/bar/pulls/1`
- ReviewedBy: Who approved the pull request before the merge. For example `Reviewed-by: Jane Doe <jane.doe@example.com>`
```

Reply:
Thanks for the suggestion, applied.